mkosi/__init__.py

```diff
@@ -3399,6 +3399,17 @@ def run_preset_all(state: MkosiState) -> None:
         run(["systemctl", "--root", state.root, "preset-all"])
 
 
+def remove_apivfs_dirs(state: MkosiState):
+    for d in ("run", "dev", "tmp", "proc", "sys"):
```

Comment:
Should `/tmp` be necesarily included here, i.e. I'm not sure I would include it among API VFS directories.